src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -495,7 +495,7 @@ private function addAnnotationsSection(ArrayNodeDefinition $rootNode)
                     ->info('annotation configuration')
                     ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('cache')->defaultValue('file')->end()
+                        ->scalarNode('cache')->defaultValue('opcache')->end()
```

Comment:
How should we name this? `opcache` is misleading IMO as we won't use the OpCacheAdapter.